### PR TITLE
Create & Add Structure of BookStore

### PR DESCRIPTION
**Pull request by MoyasiGinko**
### In this milestone, I:

- Created a new branch `react-app-setup`
- Added react components for the bookstore project
- Created bookstore structure
- Added basic styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,129 @@
...
+  <br/>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```
Comment (Munsa1) [inline README.md]: - [x] Kindly delete this link to the `Live Demo` section since we have deleted it, this link goes no where

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @MoyasiGinko, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Created Components :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,128 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to my coding partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+This project is [MIT](./LICENSE.md) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (KanzaTahreem) [inline README.md]: - [OPTIONAL] Please consider adding the section heading for _LICENSE_ for making your README more professional ⛏🤓

  image

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @MoyasiGinko 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Great work using the right Gitflow 
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (MoyasiGinko) [reply README.md]: ### Hey @KanzaTahreem 👋   
Thank you for your suggestion. I will shortly do that. 😄

Comment (Munsa1) [reply README.md]: Thanks @KanzaTahreem for that insight. I totally missed this.
Just a reminder @MoyasiGinko, for future projects all relevant headers should be present, if not this is actually a reason to invalidate the project. So please remember this :pray:

**Pull request merged**